### PR TITLE
Sverchok open subdiv 2

### PR DESCRIPTION
**Pull request by GeneralPancakeMSTR**
## Addressed problem description

#4575 [Add Catmull-Clark subdivision to Sverchok](https://github.com/nortikin/sverchok/issues/4575)

- Renames "OpenSubdiv" node to "Catmull-Clark Subdivision"
- Improved stability, particularly level 0 subdivision on topology without quads is handled a lot better, along with subdivision <= 0 in general.  

## Preflight checklist

Put an x letter in each brackets when you're done this item:

- [x] Code changes complete.
- [x] Code documentation complete.
- [x] Documentation for users complete (or not required, if user never sees these changes).
- [x] Manual testing done. 
- [ ] Unit-tests implemented.
- [x] Ready for merge.

### CODE REVIEW

**Comment by Durman** [comment]
Now you can't change `bl_idname` attribute it will break previous user layouts (present of undefined nodes) . I think fixing label is enough.

**Comment by GeneralPancakeMSTR** [comment]
> Now you can't change `bl_idname` attribute it will break previous user layouts (present of undefined nodes) . I think fixing label is enough.

Good point, I will fix that.

**Comment by GeneralPancakeMSTR** [comment]
I'm not sure the best way to do the node mute implementation. Currently I'm just mapping verts->verts and faces->faces, but this sort of breaks the Edge output if the user has that plugged in and mutes the node. I'm not sure what the best way to deal with that is, so if anyone has ideas, please let me know.

File: index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 > Failing to follow these points will break the node category parser.
 
 ## Generator
-    SvLineNodeMK4
+    SvLineNodeMK4SvOpenSubdivisionNode
```
Comment (Durman) [inline index.md:12]: Not fixed yet

**Comment by Durman** [comment]
Probably there is no need to generate edges in your case.

File: index.md (hunk 1 above)
Comment (GeneralPancakeMSTR) [reply index.md]: I did not even notice that. Thank you.

**Pull request merged**